tcf_website/views/calendar.py

```
@@ -0,0 +1,142 @@
+        {
+            "upcoming_groups": sorted_upcoming,
+            "past_groups": sorted_past,
+        },
+    )
+
+
+def event_detail(request, event_uri):
+    """Display detailed information for a specific event"""
+    try:
+        event_data = presence.get_event_details(event_uri)
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)